assets/scss/_custom.scss

```diff
@@ -806,28 +806,31 @@ body.td-documentation {
 @media (max-width: 768px) {
   #announcement {
     padding-top: 4rem;
-    padding-bottom: 4rem;
-    .announcement-main, aside .announcement-main {
-      padding-top: calc(min(2rem,2em));
-    }
-  }
-}
+    padding-bottom: 0;
+    h4{
+      display: flex;
+      flex-direction: column;
+      text-align: center;
```

Comment:
How do we know this is right (for every future announcement)?

Reply:
column flex would work well with small viewports, I'd like to know your thoughts  on this